app/lib/training_route_manager.rb

```diff
@@ -9,8 +9,19 @@ def requires_placement_details?
     feature_enabled?("routes.provider_led_postgrad") && provider_led_postgrad?
   end
 
+  def award_type
+    case trainee.training_route
+    when "early_years_undergrad"
```

Comment:
This will also be applicable to `early_years_graduate_employment_based`, `early_years_graduate_entry` and `early_years_assessment_only`. I think if you merge in master you'll see some changes in `config/initializers/training_routes` that will help you here.